ts/backend/src/app/features/controller/submission.controller.mts

```diff
@@ -396,9 +441,25 @@ export class SubmissionController extends Controller {
    *          items:
    *            type: string
    *            format: uuid
+   *    requestBody:
+   *      required: true
+   *      content:
+   *        application/json:
+   *          schema:
+   *            type: object
+   *            properties:
+   *              name:
+   *                type: string
+   *                description: Display name of submission.
+   *              code_repository:
+   *                type: string
+   *                description: URL of submission code repository.
+   *              published:
+   *                type: boolean
+   *                description: Whether submission is published.
    *    responses:
    *      200:
-   *        description: Published submission.
+   *        description: Submission patched.
```

Comment:
Expose addmissible values for `status`: https://www.speakeasy.com/openapi/schemas/enums